playground/api/v1/api.proto

```diff
@@ -81,6 +82,17 @@ message GetRunOutputResponse {
   Status compilation_status = 2;
 }
 
+// CancelRequest request to cancel code processing
+message CancelRequest {
+  string pipeline_uuid = 1;
+}
+
+// CancelResponse contains pipelineId of canceled code processing
+message CancelResponse {
+  string pipeline_uuid = 1;
```

Comment:
Is it some pattern to return the same id which is in the request? My first thought was to return nothing😁

Reply:
Changed.